.gitignore

```diff
@@ -1,5 +1,7 @@
-/build/
-/node_modules/
+**/build/
+**/node_modules/
+test/libwebrtc/
```

Comment:
disregard changes on this file please.

`test/libwebrtc` must be there.
`.vs` files are editor specific and you can set them on your profile's .gitignore.

Reply:
Will do. 
P.S. are you sure I should remove `test/libwebrtc`? It only holds the build artifacts whereas `test/deps/libwebrtc` holds the source itself

Reply:
mm, `test/libwebrtc` is not part of the repo. 

https://github.com/versatica/libmediasoupclient/tree/v3/test

Reply:
I apologize, that's my mistake. Building with the proper command places build artifacts all in `/build`, but I had tried building the tests separately which generated that directory. Removed.